### PR TITLE
[JBPM-9737] Conscious Language initiative (jBPM)

### PR DESCRIPTION
**Pull request by fjtirado**
Removing master

Exceptions 
- https://issues.redhat.com/browse/RHPAM-3697?focusedCommentId=16176154&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16176154
- https://issues.redhat.com/browse/RHPAM-3697?focusedCommentId=16176186&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16176186
- https://issues.redhat.com/browse/RHPAM-3697?focusedCommentId=16176197&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16176197

**JIRA**:

[link](https://issues.redhat.com/browse/JBPM-9737)

### CODE REVIEW

**Comment by mareknovotny** [comment]
what about other occurrences?
 like https://github.com/kiegroup/jbpm-work-items/search?q=master

**Review by mareknovotny — CHANGES_REQUESTED** [review]
please change everything like the branch and dependencies are ready to go without offended names/terms.

**Comment by fjtirado** [comment]
@mareknovotny The other occurrences  are already mentioned in the description (which I guess you missed). We need to discuss them. I cannot change the branch name in the code before the real branch name is changes or I will have an e-mail from you complaining about failed test.

**Comment by mareknovotny** [comment]
my point is that whenever we prepared all here we can change branch name and merged it after that. @mbiarnes wdyt?

**Comment by fjtirado** [comment]
@mareknovotny @lazarotti 
There are three exceptions documented here
https://issues.redhat.com/browse/BAPL-1990?focusedCommentId=16177600&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16177600
I think all of them can be accepted by PM. 
I do not think we should change branch name on GIT, or at least this is what I understood from this comment on the same bapl
https://issues.redhat.com/browse/BAPL-1990?focusedCommentId=16237547&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16237547
If finally we rename the branch name (I do not recommend that), in order to avoid failed test, I agree we should do that with this same JIRA 
For the other two exceptions, for the Karaf one I can not figure out an alternative other than not using that Karaf feature, which will probably break something, since I assume that exception is included there for a reason. Anyway, we can argue that the offensive language is in third party tool we have not control over. 
For getMasterBranch method, I think the suffix "branch" remove the offensive meaning of the word master, clearly indicating that in this context master is equivalent to main (changing that will break code using git work item)

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by fjtirado** [comment]
jenkins retest this

File: ethereum-workitem/src/test/java/org/jbpm/process/workitem/ethereum/test/contract/Greeter.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
  * <p><strong>Do not modify!</strong>
  * <p>Please use the <a href="https://docs.web3j.io/command_line.html">web3j command line tools</a>,
  * or the org.web3j.codegen.SolidityFunctionWrapperGenerator in the
- * <a href="https://github.com/web3j/web3j/tree/master/codegen">codegen module</a> to update.
+ * <a href="https://github.com/web3j/web3j">codegen module</a> to update.
```
Comment (elguardian) [inline ethereum-workitem/src/test/java/org/jbpm/process/workitem/ethereum/test/contract/Greeter.java:39]: main insteado of master

File: ethereum-workitem/src/test/java/org/jbpm/process/workitem/ethereum/test/contract/Mortal.java [hunk 2]
```diff
@@ -34,7 +34,7 @@
  * <p><strong>Do not modify!</strong>
  * <p>Please use the <a href="https://docs.web3j.io/command_line.html">web3j command line tools</a>,
  * or the org.web3j.codegen.SolidityFunctionWrapperGenerator in the
- * <a href="https://github.com/web3j/web3j/tree/master/codegen">codegen module</a> to update.
+ * <a href="https://github.com/web3j/web3j">codegen module</a> to update.
```
Comment (elguardian) [inline ethereum-workitem/src/test/java/org/jbpm/process/workitem/ethereum/test/contract/Mortal.java:37]: main instead of master

**Review by elguardian — CHANGES_REQUESTED** [review]

**Review by elguardian — APPROVED** [review]
LGTM

**Comment by elguardian** [comment]
Jenkins retest this please

**Pull request merged**